platform/ios/test/MGLNSDateAdditionsTests.mm

```diff
@@ -0,0 +1,24 @@
+#import <XCTest/XCTest.h>
+
+#include <mbgl/util/chrono.hpp>
+#import "../../darwin/src/NSDate+MGLAdditions.h"
+
+@interface MGLNSDateAdditionsTests : XCTestCase
+@end
+
+@implementation MGLNSDateAdditionsTests
+
+- (void)testDurationToNSTimeInterval {
+    
+    NSTimeInterval timeInterval = 5;
+    mbgl::Duration duration = MGLDurationFromTimeInterval(timeInterval);
+    NSTimeInterval durationTimeInterval = MGLTimeIntervalFromDuration(duration);
+    
+    mbgl::Duration durationInNanoseconds(5000000000);
```

Comment:
Since chrono.hpp imports [`<chrono>`](http://en.cppreference.com/w/cpp/chrono/operator%22%22s), you can write `using namespace std::chrono_literals;` at the top, then use `auto expectedDuration = 5000000000ns;` or even `mbgl::Duration expectedDuration = 5s;`.